otter_welcome_buddy/common/constants.py

```diff
@@ -9,10 +10,20 @@ class CronExpressions(Enum):
 
 COMMAND_PREFIX: str = "!"
 
+DATA_DIR: str = "data"
+DATA_FILE: str = "otter-buddy.db"
+DATA_FILE_PATH: str = os.path.join(DATA_DIR, DATA_FILE)
+
+ALL_DIRS = (
+    attrib_value
+    for attrib_name, attrib_value in list(globals().items())
+    if attrib_name.endswith("DIR")
+)
+
 # Discord role that give access to admin role based commands
-OTTER_ADMIN = "ROOT"
+OTTER_ADMIN: str = "ROOT"
 # Discord role that give access to moderator role based commands
-OTTER_MODERATOR = "Collaborator"
+OTTER_MODERATOR: str = "Collaborator"
```

Comment:
A moderator is a collaborator? The naming convention creates confusion, do you think that we can change it to collaborator.

Reply:
Is a convention when moderating servers, you have admins and moderators, but for our particular use case, we considered collaborators. I think we should choose if we want to let it "generic" the naming conventions (so let it as moderator) or change it to our particular use case.